monai/transforms/utility/array.py

```diff
@@ -1023,7 +1023,7 @@ def __call__(self, img: NdarrayOrTensor):
         img_np, *_ = convert_data_type(img, np.ndarray)
         img_flat = img_np.flatten()
         try:
-            out_flat = np.copy(img_flat).astype(self.dtype)
+            out_flat = np.copy(img_flat).astype(self.dtype, copy=False)
```

Comment:
I think this could be `np.array(img_flat, dtype=self.dtype)`

Reply:
Sure, updated in the latest commit.

Thanks.